src/Symfony/Component/DependencyInjection/Loader/FileLoader.php

```diff
@@ -88,8 +68,12 @@ private function findClasses($namespace, $resource)
     {
         $classes = array();
         $extRegexp = defined('HHVM_VERSION') ? '/\\.(?:php|hh)$/' : '/\\.php$/';
+        $prefixLen = null;
+        foreach ($this->glob($resource, true, $prefix) as $path => $info) {
```

Comment:
if you call glob outside of the loop, you can compute prefixLen outside of the foreach